plugin/clone/src/clone_local.cc

```diff
@@ -392,4 +392,17 @@ int Local_Callback::apply_cbk(Ha_clone_file to_file, bool apply_file,
 
   return (error);
 }
+
+[[nodiscard]] int Local_Callback::synchronize_engines() {
+  Key_Values synchronization_coordinates;
+  auto err = synchronize_logs(synchronization_coordinates);
+  if (err != 0) {
+    return err;
+  }
+  auto client = get_clone_client();
```

Comment:
References and (non-smart) pointers should be pulled up from auto:
```suggestion
  auto *client = get_clone_client();
```

Reply:
Across this file, pointers are not pulled up... keeping it consistent might be better for understanding.

Reply:
I believe that making local code better trumps consistency, especially if the surrounding code is not owned (at the moment) by us, otherwise it risks never getting improved, but it's a matter of opinion